src/reducers/index.js

```diff
@@ -1,20 +1,69 @@
+import {combineReducers} from 'redux';
+
 
 function currentCount(state=0, action){
   if(action.type === "INCREASE_COUNTER"){
-
+    return Object.assign({}, state, {
```

Comment:
Remember what currentCount is, as you can see from state.js it is just a number. 
Object.assign is only for objects.
What gets returned from currentCount must be of the same type, number. Not string, array or object.